### DISCUSSION TITLE
XQuery and XSD no QL

### DISCUSSION THREAD

**Discussion by leostuder** [sbarex/SourceCodeSyntaxHighlight#136] [Q&A]
XQuery files (.xq, .xu, .xquery) and XSD (.xsd) do not show in QL macOS monterey. Any suggestions?

<img width="552" alt="Screenshot 2022-01-02 at 23 45 46" src="https://user-images.githubusercontent.com/693382/147891428-9df214d3-05c7-43fe-b8ab-83f2956a082d.png">
<img width="552" alt="Screenshot 2022-01-02 at 23 48 33" src="https://user-images.githubusercontent.com/693382/147891467-6db71f0c-310f-4d06-bcad-5787c4a224f1.png">
<img width="552" alt="Screenshot 2022-01-02 at 23 50 44" src="https://user-images.githubusercontent.com/693382/147891508-7a51fd28-8265-4dbd-8d99-7539c34f8e47.png">

**Comment by sbarex** [comment]
Implemented on release 2.1.7.
